packages/charts-components/templates/amd-script-tag/index.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Charts components AMD template</title>
+    <style>
+        .chart-container {
+            height: 800px;
+        }
+    </style>
+
+    <link rel="icon" href="data:;base64,=" />
+
+    <!-- Load the ArcGIS Maps SDK for JavaScript -->
+    <link rel="stylesheet" href="https://js.arcgis.com/4.28/esri/themes/dark/main.css" />
+    <script src="https://js.arcgis.com/4.28/"></script>
+
+    <!-- Load the Charts Components -->
+    <script type="module" src="https://jsdev.arcgis.com/charts-components/4.29/arcgis-charts-components.esm.js"></script>
```

Comment:
we are trying not to commit jsdev to the repo, would https://js.arcgis.com/charts-components/next/arcgis-charts-components.esm.js work?

Reply:
What's wrong with https://js.arcgis.com/charts-components/4.29/arcgis-charts-components.esm.js ?
I noticed it's a few versions behind jsdev, but I assume that will be fixed later today.

Reply:
Map and coding components are also waiting on js.arcgis.com to copy our latest builds.

https://js.arcgis.com/map-components/next/builddate.txt

![image](https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/assets/49499098/319ba5fe-9e2b-4f42-b768-fdae5b98e4d1)

Reply:
ultimately we are trying to establish a `next` branch. which will always point to `next` versions of the different packages. 

Then, when there is a release. It will be a simple find & replace to switch all the urls from next to the release version.  Once this is merged we'll create the release branch and open a PR into `main`. A lot of moving parts but I think it will help manage dependencies to keep things consistent as this repo continues to grow.

I'll share an internal doc.

Reply:
Linked are now updated to `https://js.arcgis.com/charts-components/4.29/arcgis-charts-components.esm.js`